docs/docs/policy-reference/_examples/tokens/verify/sign_raw/config.json

```diff
@@ -1,6 +1,7 @@
 {
-	"showInput": false,
-	"showData":  false,
-	"showTitles": false,
-	"command": "data.jwt"
+  "showInput": false,
+  "showData": false,
+  "showTitles": false,
+  "titleSize": 5,
```

Comment:
titleSize seems to always be 5, or 2. I wonder if there's a way we can differentiate here without needing to allow arbitrary title sizes? 🤔